### ISSUE TITLE
Cephfs CSI volumes fail to attach with transport endpoint is not connected

### ISSUE THREAD

**Issue by stefanfritsch** [rook/rook#4389]
**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**

Cephfs csi volumes fail to attach. The error is `transport endpoint is not connected`. Sometimes the pod starts fine but trying to access the volume from within the pod results in the error.

**Expected behavior:**

Volumes attach successfully.

**How to reproduce it (minimal and precise):**

Hard to say as it doesn't happen with every volume every time. This started happening shortly after the upgrade to 1.1.7.

All the volumes that do have the issue are 
* cephfs-csi mounts 
* using fuse (due to bug #4055 that will hopefully be resolved in the next version).

Pods hang indefinitely at the stage where volumes are attached with an error like the following:

```
MountVolume.MountDevice failed for volume "dash-db" : stat /var/lib/kubelet/plugins/kubernetes.io/csi/pv/dash-db/globalmount: transport endpoint is not connected
```
Using `umount -f` on the volume, e.g.
```
sudo umount -f /var/lib/kubelet/plugins/kubernetes.io/csi/pv/dash-db/globalmount
```
seems to fix the issue.

I couldn't find anything related to the error in the logs of the cephfs csi provisioner.

**Environment**:
* OS (e.g. from /etc/os-release): Ubuntu 18.04.3 LTS with the hwe kernel
* Kernel (e.g. `uname -a`): 5.0.0-36-generic
* Rook version (use `rook version` inside of a Rook Pod): 1.1.7
* Storage backend version (e.g. for ceph do `ceph -v`): 14.2.4
* Kubernetes version (use `kubectl version`): v1.15.6
* Kubernetes cluster type (e.g. Tectonic, GKE, OpenShift): bare metal
* Storage backend status (e.g. for Ceph use `ceph health` in the [Rook Ceph toolbox](https://rook.io/docs/rook/master/ceph-toolbox.html)): 

```
HEALTH_WARN 1 subtrees have overcommitted pool target_size_bytes; 1 subtrees have overcommitted pool target_size_ratio
```
(I assume the warning is due to [this](https://www.spinics.net/lists/ceph-users/msg52606.html) issue)

**Comment by Madhu-1** [comment]
https://github.com/ceph/ceph-csi/issues/703

**Comment by stefanfritsch** [comment]
Sorry, but if that's the case you need to put a warning in the release notes and/or amend the upgrade docs from

> Unless otherwise noted due to extenuating requirements, upgrades from one patch release of Rook to another are as simple as updating the image of the Rook operator. For example, when Rook v1.1.7 is released, the process of updating from v1.1.0 is as simple as running the following:

to include

> If you use the FUSE plugin to mount volumes (<#insert steps how to find out if you do; as mentioned I use the fuse plugin even though forcing the kernel driver due to version bug>) you will have to reboot all nodes after the upgrade to remove stale mounts (<#insert steps to safely reboot a node>)

**Comment by Madhu-1** [comment]
Fixed by https://github.com/rook/rook/pull/4496
